bndtools.m2e/src/bndtools/m2e/MavenRunListenerHelper.java

```diff
@@ -23,15 +23,18 @@
 import org.eclipse.core.resources.ResourcesPlugin;
 import org.eclipse.core.runtime.CoreException;
 import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
 import org.eclipse.core.runtime.NullProgressMonitor;
+import org.eclipse.core.runtime.Status;
 import org.eclipse.m2e.core.MavenPlugin;
 import org.eclipse.m2e.core.embedder.IMaven;
 import org.eclipse.m2e.core.project.IMavenProjectFacade;
 import org.eclipse.m2e.core.project.IMavenProjectRegistry;
+import org.eclipse.ui.statushandlers.StatusManager;
```

Comment:
hey @juergen-albert  this dependency on the `org.eclipse.ui` import is what is breaking the build in the PR.  We haven't yet had a dependency from `bndtools.m2e` onto a UI plugin.  We should probably remove this reference for something and instead use something from `org.eclipse.core`  or just do nothing at all (return null silently)

Reply:
The Idiot I am, I forgot to commit the bnd file...

The StatusManager is used to show a message to the user, to tell him what the issue is. Without it, resolving would not work and one has no clue what the issue is.

Reply:
I would like to propose a different way, other than having bndtools.m2e rely on UI apis.  This method `getMojoExecution` already throws CoreException.  `CoreException` can have a Status object where you can put your message.  Then further up the call stack, we can have CoreException be caught by a bndtools plugin that already interacts with the UI, namely, `LaunchUtils.createRun`.  That method could then catch the CoreException and grab the status and call the StatusHandler.handle() method.

Reply:
good point!